docs/concepts/tokens/decentralized-exchange/autobridging.md

```diff
@@ -11,10 +11,13 @@ labels:
 
 Any [Offer](offers.md) in the XRP Ledger's [decentralized exchange](index.md) that would exchange two tokens could potentially use XRP as an intermediary currency in a synthetic order book. This is because of _auto-bridging_, which serves to improve liquidity across all asset pairs by using XRP when doing so is cheaper than trading directly. This works because of XRP's nature as a native cryptocurrency to the XRP Ledger. Offer execution can use a combination of direct and auto-bridged offers to achieve the best total exchange rate.
 
-Example: _Anita places an offer to sell GBP and buy BRL. She might find that this uncommon market has few offers. There is one offer with a good rate, but it has insufficient quantity to satisfy Anita's trade. However, both GBP and BRL have active, competitive markets to XRP. The XRP Ledger's auto-bridging system finds a way to complete Anita's offer by purchasing XRP with GBP from one trader, then selling the XRP to another trader to buy BRL. Anita automatically gets the best rate possible by combining the small offer in the direct GBP:BRL market with the better composite rates created by pairing GBP:XRP and XRP:BRL offers._ <!-- SPELLING_IGNORE: gbp -->
+Example: _Anita places an offer to sell EUR and buy USD. She might find that this uncommon market has few offers. There is one offer with a good rate, but it has insufficient quantity to satisfy Anita's trade. However, both EUR and USD have active, competitive markets to XRP. The XRP Ledger's auto-bridging system finds a way to complete Anita's offer by purchasing XRP with EUR from one trader, then selling the XRP to another trader to buy USD. Anita automatically gets the best rate possible by combining the small offer in the direct EUR:USD market with the better composite rates created by pairing EUR:XRP and XRP:USD offers._ 
```

Comment:
This paragraph doesn't carry quite the same meaning if you substitute the currencies for different ones; the point of the original wording is that GBP:BRL is not a very active market so it can benefit from XRP as a bridge currency. The EUR:USD market, which is one of the most actively-traded currency pairs in the world, does not benefit nearly as much.

If possible, it would be better to change the diagram than change the paragraph.

Reply:
Absolutely! I agree, will change accordingly

Reply:
@mDuo13 done :)